web2/htdocs/js/events.js

```diff
@@ -165,6 +165,87 @@
             });
           });
       }) /* }}} */
+      /* }}} */
+
+      /* "Pause Job" links (href="pause:...") */
+      .on('click', 'a[href^="pause:"], button[rel^="pause:"]', function (event) { /* {{{ */
+        event.preventDefault();
+        var uuid;
+        if ($(event.target).is('button')) {
+          uuid = $(event.target).attr('rel');
+        } else {
+          uuid  = $(event.target).closest('a[href^="pause:"]').attr('href');
+        }
+        uuid = uuid.replace(/^pause:/, '');
+
+        api({
+          type: 'POST',
+          url:  '/v2/tenants/'+AEGIS.current.uuid+'/jobs/'+uuid+'/pause',
+          success: function () {
+            banner('Pausing');
+          },
+          error: function () {
+            banner('unable to pause', 'error');
+          }
+        });
+      }) /* }}} */
+      /* }}} */
+
+      /* "Unpause Job" links (href="unpause:...") */
+      .on('click', 'a[href^="unpause:"], button[rel^="unpause:"]', function (event) { /* {{{ */
+        event.preventDefault();
+        var uuid;
+        if ($(event.target).is('button')) {
+          uuid = $(event.target).attr('rel');
+        } else {
+          uuid  = $(event.target).closest('a[href^="unpause:"]').attr('href');
+        }
+        uuid = uuid.replace(/^unpause:/, '');
+
+        api({
+          type: 'POST',
+          url:  '/v2/tenants/'+AEGIS.current.uuid+'/jobs/'+uuid+'/unpause',
+          success: function () {
+            banner('Unpausing');
+          },
+          error: function () {
+            banner('unable to unpause', 'error');
+          }
+        });
+      }) /* }}} */
+      /* }}} */
+
+      /* "Delete Job" links (href="delete:...") */
+      .on('click', 'a[href^="delete:"], button[rel^="delete:"]', function (event) { /* {{{ */
+        event.preventDefault();
+        var uuid;
+        if ($(event.target).is('button')) {
+          uuid = $(event.target).attr('rel');
+        } else {
+          uuid  = $(event.target).closest('a[href^="delete:"]').attr('href');
+        }
+        uuid = uuid.replace(/^delete:/, '');
+        var job = AEGIS.job(uuid);
+        var name = job.name
+
+        modal($.template('delete-are-you-sure', {
+            uuid: uuid,
+            name: name
```

Comment:
You can greatly simplify this whole section by just passing the job to the template, directly.

i.e.:

```
modal($.template('delete-are-you-sure', AEGIS.job(uuid.replace(/^delete:/, ''))))
```

or, if you prefer, keep the UUID cleaning from line 227, but don't limit the modal to just name / uuid; what if we want to show the schedule (is that the daily 3am job or the daily 3:05am?)?